spec/unit/settings_spec.rb

```diff
@@ -1308,7 +1308,7 @@ def assert_accessing_setting_is_deprecated(settings, setting)
       @settings.to_catalog
     end
 
-    describe "on Microsoft Windows" do
+    describe "on Microsoft Windows", :if => Puppet.features.microsoft_windows? do
```

Comment:
The original commit adding this test was https://github.com/hlindberg/puppet/commit/47058abc0c5647d59b0dd21181e67dbfdd908292 from @joshcooper 

If you're going to add the `:if => Puppet.features.microsoft_windows?` line here, then (which I think is OK given we have more robust testing now), then you can remove the stub line below:

```ruby
Puppet.features.stubs(:microsoft_windows?).returns true
```